src/text-filter/internal.tsx

```diff
@@ -44,16 +51,15 @@ const InternalTextFilter = React.forwardRef(
           value={filteringText}
           disabled={disabled}
           autoComplete={false}
+          ariaDescribedby={showResults ? liveContentId : undefined}
           clearAriaLabel={filteringClearAriaLabel}
           onChange={event => fireNonCancelableEvent(onChange, { filteringText: event.detail.value })}
           __onDelayedInput={event => fireNonCancelableEvent(onDelayedChange, { filteringText: event.detail.value })}
         />
-        <span
-          aria-live="polite"
-          aria-atomic="true"
-          className={clsx(styles.results, showResults && styles['results-visible'])}
-        >
-          {showResults ? countText : ''}
+        <span className={clsx(styles.results, showResults && styles['results-visible'])}>
+          <LiveRegion delay={LIVE_REGION_DELAY} visible={true}>
+            <span id={liveContentId}>{showResults ? countText : ''}</span>
```

Comment:
In case there are no results, the `aria-describedby` is `undefined`, but the inner span is still rendering empty, as caught by this test: https://github.com/cloudscape-design/components/actions/runs/4242559394/jobs/7374220883

```
FAIL src/s3-resource-selector/s3-modal/__tests__/buckets-table.test.tsx
  ● filtering buckets

    expect(element).toBeEmptyDOMElement()

    Received:
      "<span><span id=\"text-filter79-1677068421820-9802\"></span></span><span class=\"awsui_root_xttbq_1i4dh_93 awsui_root_3bgfn_im8v7_5\"><span aria-atomic=\"true\" aria-live=\"polite\"></span></span>"
```

Is the inner span with the `id` necessary when there are no results? If so, `aria-describedby` should probably still be defined; if not, we can apply the conditional render to the entire span.

Reply:
Sounds good, but then if the span is always rendered, does it make sense to make the `aria-describedby` attribute conditional or should line 54 better be just like

```
ariaDescribedby={liveContentId}
```